manifests/rules/samba.pp

```diff
@@ -0,0 +1,22 @@
+# @summary manage Samba, the suite to allow Windows file sharing on Linux resources.
+# @param ctdb Enable ctdb-driven clustered Samba setups.
+class nftables::rules::samba (
+  Boolean $ctdb = false,
+) {
+  nftables::rule {
+    'default_in-netbios_tcp':
+      content => 'tcp dport {139,445} accept',
+  }
+
+  nftables::rule {
+    'default_in-netbios_udp':
+      content => 'udp dport {137,138} accept',
+  }
+
+  if ($ctdb) {
```

Comment:
imho there is not need for the parantheses

Reply:
indeed, that was a left-over - removed now